scylla-server/src/index.ts

```diff
@@ -20,6 +20,18 @@ app.get("/runs", async (req: Request, res: Response) => {
   }
 });
 
+// #15 CRUD Operations 
+// Service Function, gets all systems and returns result as json
+app.get("/systems", async (req: Request, res: Response) => {
```

Comment:
ok so this would be great if we were still using http. But now that were using a socket we actually just need the prisma.system.findMany() call in an exported function in  a system.services.ts file in a services folder.

Reply:
ok i've updated so it should work for websocket